rclcpp_action/include/rclcpp_action/types.hpp

```diff
@@ -15,8 +15,8 @@
 #ifndef RCLCPP_ACTION__TYPES_HPP_
 #define RCLCPP_ACTION__TYPES_HPP_
 
+#include <array>
 #include <climits>
-#include <functional>
```

Comment:
`std::hash<T>` transitively gets included here, but shouldn't this header stay to explicitly include the template?

Reply:
Oh, I see.  I didn't realize that `std::less` and `std::hash` are included by putting them into namespace `std`.  Technically, we don't actually need to have the `#include` in here, since we are generating the code right here and we don't depend on anything else.  But I guess it would be nicer to include the template specialization for `std::hash` and `std::less`, so I'll add that back.  Thanks for the heads up.

Reply:
Fixed in 1492cd9